### PR TITLE
hyperlinked eCapris id

### PR DESCRIPTION
**Pull request by tillyw**
Fixes https://github.com/cityofaustin/atd-data-tech/issues/5199

Adds eCapris subproject ID hyperlinks to project list view and project summary view. Links tested in eCapris and working! Also, they open in a new tab.

Screen Shot 2021-02-11 at 7 08 38 PM

Screen Shot 2021-02-11 at 7 08 18 PM

### CODE REVIEW

**Review by mateoclarke — CHANGES_REQUESTED** [review]
🚢  💨 

Looks good to me on the link side of things. I added @amenity for her review. For Moped, we're trying to include product acceptance within our Pull Request process.

I was wondering why the 3 lock files got updated. Seems like we only need `moped-editor/package-lock.json` update. The other two must've been generated by running `npm install` commands in subdirectories that don't need it. Can you delete those two updates?

**Review by mateoclarke — APPROVED** [review]
👍

**Comment by amenity** [comment]
Works beautifully, @tillyw! I had [specifications about design](https://github.com/cityofaustin/atd-data-tech/issues/5199) in the issue here. Do you want those to go in a separate issue?

File: moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js [hunk 1]
```diff
@@ -113,6 +116,12 @@ export const ProjectsListViewQueryConf = {
       searchable: true,
       sortable: false,
       label: "eCapris Subp.",
+      filter: value =>
```
Comment (johnclary) [inline moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js:119]: cool to see how this plugs into the filter component!

Comment (tillyw) [reply moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js:118]: thanks, John!

**Comment by tillyw** [comment]
Hey y'all, I pushed a couple new commits, which 1) move the pre-existing code to a reusable "ExternalLink" component that takes in a url and some text to render (as per @mateoclarke's direction), and 2) adhere to @amenity's design specifications (adding default url formatting and behavior and external link icon from material ui). Please let me know if the formatting looks okay and, if so, if we're good to go on this one!

Screen Shot 2021-02-25 at 11 35 46 AM

https://user-images.githubusercontent.com/35410637/109193104-a4792a00-775d-11eb-9d1d-139d4434eb9e.mov

**Comment by tillyw** [comment]
Actually it looks like code that was merged to main in the past hour or so creates a ton of conflicts and I'm going to have to rework this code substantially. Will keep you posted @mateoclarke & @amenity.

**Comment by tillyw** [comment]
Okay, y'all, this is working fully with the new codebase! Ready for final review @amenity @mateoclarke. Thanks for the help @sergiogcx.

**Comment by sergiogcx** [comment]
> Okay, y'all, this is working fully with the new codebase! Ready for final review @amenity @mateoclarke. Thanks for the help @sergiogcx.

Happy to help! :)

**Comment by amenity** [comment]
😍  styles are perfect! I am seeing the external link icon on all rows on the Projects table. 🤔
Screen Shot 2021-02-26 at 4 22 24 PM

**Comment by tillyw** [comment]
> 😍 styles are perfect! I am seeing the external link icon on all rows on the Projects table. 🤔
> Screen Shot 2021-02-26 at 4 22 24 PM

Thanks for pointing that out, @amenity! For some reason it wasn't rendering that way locally for me, but it makes sense that this would happen! I added some conditional logic to render a dash when there is no eCapris ID associated with a project. I also improved the styling a bit to keep what is rendered on the same line as the pencil icon (when there's room).

Screen Shot 2021-03-02 at 10 45 03 AM
Screen Shot 2021-03-02 at 10 45 28 AM
Screen Shot 2021-03-02 at 10 45 37 AM

**Review by amenity — APPROVED** [review]
💯 ❤️

**Pull request merged**